runelite-client/src/main/java/net/runelite/client/plugins/friendlist/FriendListPlugin.java

```diff
@@ -109,6 +124,15 @@ else if (event.getScriptId() == ScriptID.IGNORE_UPDATE)
 		}
 	}
 
+	@Subscribe
+	public void onGameTick(GameTick gameTick)
+	{
+		if (config.showTobIgnores())
+		{
+			colorIgnoredPlayersTob(config.showIgnoresColor());
+		}
+	}
```

Comment:
This can be replaced with a `PostScriptFired` subscriber for the scripts which build these interfaces instead of running every tick.